gammapy/data/counts_spectrum.py

```diff
@@ -0,0 +1,188 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (print_function)
+
+import numpy as np
+from astropy import log
+from ..spectrum.energy import *
+import datetime
+from astropy.io import fits
+
+__all__ = ['CountsSpectrum']
+
+class CountsSpectrum(object):
+    """Counts spectrum dataset
+
+    Parameters
+    ----------
+
+    counts: `~numpy.array`, list
```

Comment:
This won't render correctly in the Sphinx HTML docs.
There needs to be a space before the colon:
```
counts : ...
```
There's a few other places where this needs to be fixed.
I always check the HTML version of the docstrings I write ... need to find a way to make such formatting errors make the Sphinx build fail on travis-ci.